Live-Snap/Live-Snap/Views/SaveSuccessPopUpView.swift

```diff
@@ -0,0 +1,141 @@
+//
+//  SaveSuccessPopUpView.swift
+//  Live-Snap
+//
+//  Created by Oleg Abalonski on 5/28/18.
+//  Copyright © 2018 Oleg Abalonski. All rights reserved.
+//
+
+import UIKit
+
+class SaveSuccessPopUpView: UIView {
+    
+    var successIconImageView: UIImageView!
+    var successLabel: UILabel!
+    var lineBreakOneView: UIView!
+    var goToPhotosImageView: UIImageView!
+    var goToPhotosIconButton: UIButton!
+    var lineBreakTwoView: UIView!
+    var lineBreakThreeView: UIView!
+    var newImageView: UIImageView!
+    var newIconButton: UIButton!
+    var rateImageView: UIImageView!
+    var rateIconButton: UIButton!
+    
+    override init(frame: CGRect) {
+        super.init(frame: frame)
+        setLayout()
+    }
+    
+    required init?(coder aDecoder: NSCoder) {
+        fatalError("init(coder:) has not been implemented")
+    }
+    
+    func setLayout() {
+        backgroundColor = UIColor.snapBlack
+        layer.cornerRadius = 5
+        layer.masksToBounds = true
+        
+        successIconImageView = UIImageView(frame: CGRect(x: 0.0, y: 0.0, width: 40, height: 40))
```

Comment:
in this setLayout you initialize your views here, but in the icon button setLayout you initialize the views in the init. Choose one way of doing it and be consistent everywhere